examples/hello-world/package.json

```diff
@@ -16,6 +21,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build"
+    "build": "react-scripts build",
+    "translate": "cross-env NODE_ENV=production babel ./src > NUL"
```

Comment:
I see you have the translations folder, check you have examples/hello-world/translations/messages/src/App.json